GenderPayGap.WebUI/Controllers/Admin/AdminDataMigrationController.cs

```diff
@@ -0,0 +1,648 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.IO.Compression;
+using System.Linq;
+using System.Net.Http;
+using System.Text;
+using System.Threading;
+using Autofac;
+using GenderPayGap.Core;
+using GenderPayGap.Core.Interfaces;
+using GenderPayGap.Database;
+using GenderPayGap.Database.Models;
+using GenderPayGap.WebUI.Helpers;
+using GenderPayGap.WebUI.Models.Admin;
+using GovUkDesignSystem.Parsers;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using Newtonsoft.Json;
+
+namespace GenderPayGap.WebUI.Controllers
+{
+    [Authorize(Roles = LoginRoles.GpgAdmin)]
+    [Route("admin")]
+    public class AdminDataMigrationController : Controller
+    {
+
+        private readonly IDataRepository dataRepository;
+
+        public AdminDataMigrationController(IDataRepository dataRepository)
+        {
+            this.dataRepository = dataRepository;
+        }
+
+
+        [HttpGet("data-migration/export-all")]
+        public IActionResult ExportAllData()
+        {
+            if (dataRepository.GetAll<User>().Count() > 0)
```

Comment:
Why? 😃

Reply:
Similarly, this was an earlier version of the code added as a merge error - removed now.
This logic has now been moved into the better-named `IsUserAllowedToImportData`